test/mingap_test.cpp

```diff
@@ -80,18 +80,32 @@ TEST(MinGap, SphereSphereOutOfBounds) {
 
 TEST(MinGap, ClosestPointOnEdge) {
   auto a = Manifold::Cube();
-  auto b = Manifold::Sphere(1.0f).Translate({3.0f, 0.0f, 0.5f});
+  auto b = Manifold::Cube().Translate({2.0f, 1.0f, 0.5f});
```

Comment:
The closest approach here comes at multiple locations including at a vertex. To test edge-to-edge you'll need to apply rotation as well so that the unique closest point comes in the interior of two edges.

Reply:
Fixed, [this](https://manifoldcad.org/#EdgeEdge%3Ccode%3E%2F%2F%20Write%20code%20in%20JavaScript%20or%20TypeScript%20and%20this%20editor%20will%20show%20the%0A%2F%2F%20API%20docs.%20Type%20e.g.%20%22box.%22%20to%20see%20the%20Manifold%20API.%20Type%0A%2F%2F%20%22CrossSection.%22%20or%20%22Manifold.%22%20to%20list%20the%202D%20and%203D%20constructors%2C%0A%2F%2F%20respectively.%20Type%20%22module.%22%20to%20see%20the%20static%20API%20-%20these%20functions%0A%2F%2F%20can%20also%20be%20used%20bare.%20Use%20console.log()%20to%20print%20output%20(lower-right).%0A%2F%2F%20This%20editor%20defines%20Z%20as%20up%20and%20units%20of%20mm.%0Aconst%20%7Bcube%2C%20sphere%7D%20%3D%20Manifold%3B%0Aconst%20s%20%3D%20cube().rotate(%5B0%2C%200%2C%2045%5D)%3B%0Aconst%20s2%20%3D%20cube().rotate(%5B0%2C%2045%2C%200%5D).translate(%5B1%2C%200%2C%200.5%5D)%0A%0A%0Aconst%20result%20%3D%20s2.add(s)%3B) should do it!